src/DIRAC/DataManagementSystem/DB/FileCatalogDB.py

```diff
@@ -969,12 +969,16 @@ def getDirectoryReplicas(self, lfns, allStatus, credDict):
     successful = res['Value']['Successful']
     return S_OK({'Successful': successful, 'Failed': failed})
 
-  def getDirectorySize(self, lfns, longOutput, fromFiles, credDict):
+  def getDirectorySize(self, lfns, longOutput, fromFiles, recursiveSum, credDict):
```

Comment:
Isn't it better to do

```suggestion
  def getDirectorySize(self, lfns, longOutput, fromFiles, credDict, recursiveSum):
```
?

Reply:
in a publicly exposed API, yes, but this is all internal, so for consistency with all the other methods, I'd rather keep the `credDict` as last parameter